app/assets/javascripts/discourse/app/lib/lazy-load-images.js

```diff
@@ -15,7 +15,15 @@ export function nativeLazyLoading(api) {
       forEachImage(post, (img) => {
         img.loading = "lazy";
         if (img.dataset.smallUpload) {
-          img.style = `background-image: url(${img.dataset.smallUpload}); background-size: cover;`;
+          if (!img.complete) {
+            if (!img.onload) {
+              img.onload = () => {
+                img.style = "";
```

Comment:
Can we use `removeAttribute("style")` on the element? Would have cleaner markup, instead of many empty `style` properties.